controllers/controllers.js

```diff
@@ -51,4 +51,20 @@ exports.getArticleComments = (req, res, next) =>{
    })
    
    
+}
+
+exports.postComment = (req, res, next) =>{
+    const {article_id} = req.params;
+    const comment = req.body
+    //check article exists
+    //check user exists
+    const doesArticleExist= checkArticleExists(article_id)
+    const doesUserExist  = checkUserExists(comment.username)
```

Comment:
This one isn't needed. If you look at the seed data for this you'll see a foreign key constraint. That means you'll get a psql error if trying to post as a user that doesn't exist.